src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsParameters.java

```diff
@@ -32,23 +32,20 @@
 //END_TIGER_EXCLUDE
 
 /**
- * Represents the set of parameters for each https
- * connection negotiated with clients. One of these
- * is created and passed to
- * {@link HttpsConfigurator#configure(HttpsParameters)}
- * for every incoming https connection,
- * in order to determine the parameters to use.
- * <p>
- * The underlying SSL parameters may be established either
- * via the set/get methods of this class, or else via
- * a {@link javax.net.ssl.SSLParameters} object. SSLParameters
- * is the preferred method, because in the future,
- * additional configuration capabilities may be added to that class, and
- * it is easier to determine the set of supported parameters and their
- * default values with SSLParameters. Also, if an SSLParameters object is
- * provided via
- * {@link #setSSLParameters(SSLParameters)} then those parameter settings
- * are used, and any settings made in this object are ignored.
+ * Represents the set of parameters for each https connection negotiated with
+ * clients. One of these is created and passed to
+ * {@link HttpsConfigurator#configure(HttpsParameters)} for every incoming https
+ * connection,in order to determine the parameters to use.
+ *
```

Comment:
looks like a space is missing between "connection,in"

Reply:
Typo fixed in https://github.com/openjdk/jdk/pull/610/commits/45f451829c30339a10f03377a78b17b7291be82f